README.md

```diff
@@ -1,15 +1,72 @@
 # Submariner Operator
 
-The submariner operator installs the submariner components on a kubernetes cluster.
+**Project status**: The project is meant only to be used as a development preview. Installing the operator on an Openshift cluster may disable some of the operator features. 
+
+The submariner operator installs the submariner components on a Kubernetes cluster.
 
 It's available on [OperatorHub:submariner](https://operatorhub.io/operator/submariner).
 
-The current release is very basic, and will deploy the submariner components
-in your cluster, but it does not deploy the broker yet.
+# Quickstart
+
+
+## Prerequisites
+
+Submariner has a few requirements to get started:
+
+- At least 2 Kubernetes clusters, one of which is designated to serve as the central broker that is accessible by all of your connected clusters; this can be one of your connected clusters, but comes with the limitation that the cluster is required to be up to facilitate interconnectivity/negotiation
+- Different cluster/service CIDR's (as well as different Kubernetes DNS suffixes) between clusters. This is to prevent traffic selector/policy/routing conflicts.
+- Direct IP connectivity between instances through the internet (or on the same network if not running Submariner over the internet). Submariner supports 1:1 NAT setups but has a few caveats/provider-specific configuration instructions in this configuration.
+- Knowledge of each cluster's network configuration
+- Worker node IPs on all the clusters must be outside of the cluster/service CIDR ranges.
+- kubefedctl installed ([0.1.0-rc3](https://github.com/kubernetes-sigs/kubefed/releases/tag/v0.1.0-rc3)).
```

Comment:
I would mark the kubefedctl and / kubectl as requisites for *only* the discovery part.

Reply:
Done.